index.html

```diff
@@ -329,7 +340,15 @@ <h2>Contact me</h2>
             </div>
         </section>
     </div>
+    <script>
+        function openNav() {
+            document.getElementById("myNav").style.display = "block";
+        }
 
+        function closeNav() {
+            document.getElementById("myNav").style.display = "none";
+        }
+    </script>
```

Comment:
- [ ] It's also not a good practice for you javascript code to be inside the HTML,